config/components/rbac/pending_workloads_viewer_role.yaml

```diff
@@ -0,0 +1,17 @@
+# permissions for end users to view pending workloads.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: pending-workloads-viewer-role
+  labels:
+    rbac.kueue.x-k8s.io/batch-admin: "true"
+    rbac.kueue.x-k8s.io/batch-user: "true"
+rules:
+- apiGroups:
+  - visibility.kueue.x-k8s.io
+  resources:
```

Comment:
should we have more granular permissions since we introduce support for subresources? In particular, the name of the file refers to pending workloads, but not its contents.

Reply:
I've narrowed the permissions of this role